js_modules/dagster-ui/packages/ui-core/src/runs/Run.tsx

```diff
@@ -309,7 +310,9 @@ const RunWithData: React.FC<RunWithDataProps> = ({
                 counts={logs.counts}
               />
               {logType !== LogType.structured ? (
-                supportsCapturedLogs ? (
+                !computeLogFileKey ? (
+                  <NoStepSelectionState type={logType} />
```

Comment:
Just calling out that it seems like this will appear as the default view now? No logs shown until you click a step?

Reply:
Yeah it seems that was specifically asked for in the figma https://www.figma.com/file/OnaH8ynIsPg3yh7BpwyDQX/Logging-Improvements?type=design&node-id=11-19&mode=design&t=OEqHt51q5xRoPvXh-0

Reply:
yeah, i think this is actually preferred to help clarify that each step has it's own raw logs